### PR TITLE
feat(navbar): various navbar spacing tweaks

### PR DESCRIPTION
**Pull request by andipaetzold**
# Purpose of PR

This PR makes the navbar more consistent, also changes the spacing/sizes to be closer to the designs.

General:
- All navbars have a height of 60px now

Regular navbar
- All items have a total height of 28px. (Avatar has 24px, but will have 28px with the planned border)
- Fixed padding-right for icon-items with menus
- Icon buttons are closer together in the designs. But wit the 44px click size, we can't bring them any closer
- Icon buttons are square now

Loading
- Increased skeleton text height

Mobile menu button:
- Reduced height to 36px
- Reduced horizontal padding

It's best to check chromatic for the detailed changes

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/components/navbar/src/NavbarSwitcher/NavbarSwitcherSkeleton.tsx
Comment (cf-remylenoir) [inline packages/components/navbar/src/NavbarSwitcher/NavbarSwitcherSkeleton.tsx:1]: Not a biggie, but the `NavbarSwitcherSkeleton` is focusable and does not expand to the fullest available width.

Screenshot 2024-08-30 at 14 50 33

Comment (andipaetzold) [reply packages/components/navbar/src/NavbarSwitcher/NavbarSwitcherSkeleton.tsx:1]: Will fix the focus

Should it expand? Or rather be aligned to the right?

Comment (andipaetzold) [reply packages/components/navbar/src/NavbarSwitcher/NavbarSwitcherSkeleton.tsx:1]: Double checking - I even reduced the radius 🤷

**Review by cf-remylenoir — APPROVED** [review]
Thank you for taking care of this 👍

**Pull request merged**